### ISSUE TITLE
Fallback for `screen-reader-text`

### ISSUE THREAD

**Issue by stian-overasen** [DekodeInteraktiv/savage-cards#47]
The `screen-reader-text` attribute value for the link component should have fallback values if link text is not set.

Possible values:
- Card more text
- Card title override
- Card title

https://github.com/DekodeInteraktiv/savage-cards/blob/master/components/link.php#L28
